package-lock.json

```diff
@@ -4238,6 +4239,21 @@
         "@opentelemetry/api": "^1.0.0"
       }
     },
+    "experimental/packages/otlp-exporter-base/node_modules/@opentelemetry/otlp-transformer/node_modules/@opentelemetry/core": {
```

Comment:
There is a vestigial entry in here to clean up. I *think* just manually editing it away will be fine. Otherwise the only way I've been able to clean out some entries like this before was to:

```
cd experimental/package/otlp-exporter-base
npm uninstall @opentelemetry/otlp-transformer
npm install @opentelemetry/otlp-transformer
```